src/components/SQFormDialogStepper/SQFormDialogStepper.js

```diff
@@ -184,35 +205,41 @@ export function SQFormDialogStepper({
             <DialogTitle disableTypography={true}>
               <Typography variant="h4">{title}</Typography>
             </DialogTitle>
-            <DialogContent
-              dividers={true}
-              style={{...contentStyle, paddingTop: '.75rem'}}
-            >
-              <div className="SQFormDialogStepper__stepContainer">
+            <Divider />
+            <div className="SQFormDialogStepper__stepContainer">
+              <Grid container className="SQFormDialogStepper__stepper">
                 <IconButton
+                  height={'80px'}
+                  width={'80px'}
                   title="Previous Step"
                   IconComponent={ArrowLeftIcon}
                   isDisabled={activeStep === 0}
                   isIconTeal={true}
                   onClick={handleBack}
                 />
-                <div className="SQFormDialogStepper__stepper">
-                  <Stepper nonLinear activeStep={activeStep}>
-                    {steps.map((child, index) => (
-                      <Step key={child.props.label}>
-                        <StepButton
-                          onClick={handleStep(index)}
-                          completed={completed[index]}
-                        >
-                          {child?.props.label}
-                        </StepButton>
-                      </Step>
-                    ))}
-                  </Stepper>
-                </div>
+                <Stepper nonLinear activeStep={activeStep} classes={classes}>
+                  {steps.map((child, index) => (
+                    <Step key={child.props.label}>
+                      <StepButton
+                        onClick={handleStep(index)}
```

Comment:
This function is going to be executed on render right?
If i'm reading the code right this should be `onClick={() => handleStep(index)}`

Reply:
@SeanGroff No it only fires when you click on one of the steps in the breadcrumb.  I just tested it with some console logs.

Reply:
@SeanGroff I thought the same thing, handleStep returns a function

Reply:
> @SeanGroff I thought the same thing, handleStep returns a function

Ohhhhhh ok, so we are partially applying the function with the index. That makes sense then!